### DISCUSSION TITLE
Dealing with journals that require a public dataset prior to receiving a doi

### DISCUSSION THREAD

**Discussion by timmccubbin** [PRIDE-Archive/pride-resources#37] [General]
Dear PRIDE team,

We repeatedly deal with journals that will accept a paper for publication but require all data to be publicaly released prior to receiving our proofs and a doi. This doesn't quite fit with the PRIDE workflow where a doi is requested prior to making it public, after which we can't alter the record. In this case, do we need to just publish, receive the doi and contact you directly to add the doi? 

Kind regards,

Tim

**Comment by ypriverol** [comment]
Dear @timmccubbin you can check the following discussion https://github.com/orgs/PRIDE-Archive/discussions/14 where we explain how to publish a dataset without DOI or PUBMED Id from your dashboard.
